deploy/charts/README.md

```diff
@@ -111,60 +111,60 @@ helm install openebs-monitoring openebs-monitoring/monitoring --namespace openeb
 | `kube-prometheus-stack.grafana.plugins`                                                   | Adds Grafana plugin list to be installed                                            | `["grafana-polystat-panel","snuids-trafficlights-panel"]` |
-| `openebsMonitoringAddon.cStor.enabled`                                                          | Enables installation of monitoring addons for cStor                      | `true`                                                    |
-| `openebsMonitoringAddon.cStor.dashboards.enabled`                                                          | Enables dashboard installation of openebs cStor Storage engine                      | `true`                                                    |
-| `openebsMonitoringAddon.jiva.enabled`                                                          | Enables installation of monitoring addons for Jiva                      | `true`                                                    |
-| `openebsMonitoringAddon.jiva.dashboards.enabled`                                                           | Enables dashboard installation of openebs Jiva Storage engine                       | `true`                                                    |
-| `openebsMonitoringAddon.localPV.enabled`                                                          | Enables installation of monitoring addons for localPV                      | `true`                                                    |
-| `openebsMonitoringAddon.lvmLocalPV.enabled`                                                          | Enables installation of monitoring addons for lvm-LocalPV                      | `true`                                                    |
-| `openebsMonitoringAddon.localPV.dashboards.enabled`                                                        | Enables dashboard installation of openebs Local PV Storage engine                   | `true`                                                    |
-| `openebsMonitoringAddon.lvmLocalPV.dashboards.enabled`                                                     | Enables dashboard installation of openebs lvm-localpv storage engine                | `true`                                                    |
-| `openebsMonitoringAddon.npd.dashboards.enabled`                                                            | Enables dashboard installation related to node problems                             | `true`                                                    |
-| `openebsMonitoringAddon.ndm.dashboards.enabled`                                                            | Enables dashboard installation of OpenEBS node-disk-manager(NDM)                    | `true`                                                    |
+| `openebsMonitoringAddon.cStor.enabled`                                                    | Enables installation of monitoring addons for cStor                                 | `true`                                                    |
```

Comment:
Can the unplanned whitespace changes be removed?

Reply:
Formatting the file adds these.